doc/source/whatsnew/v0.17.0.txt

```diff
@@ -39,6 +39,35 @@ Backwards incompatible API changes
 Other API Changes
 ^^^^^^^^^^^^^^^^^
 
+- ``DataFrame.convert_objects`` keyword arguments have been shortened:
+
+=====================   =============
+Old                     New
+=====================   =============
+``convert_dates``       ``datetime``
+``convert_numeric``     ``numeric``
+``convert_timestamp``   ``timestamp``
+=====================   =============
+
+- Coercing types with ``DataFrame.convert_objects`` is now implemented using the
+keyword argument ``coerce=True``.  Previously types were coerced by setting a
+keyword argument to ``'coerce'`` instead of ``True``, as in ``convert_dates='coerce'``.
+
+  Previously data was coerced to a common dtype before serialisation, which for
+  example resulted in integers being serialised to floats:
+
+  .. code-block:: python
+
+    In [2]: df = pd.DataFrame({'i': ['1','2'], 'f': ['apple', '4.2']})
+    In [3]: df.convert_objects(convert_numeric='coerce')
+
+  Now the ``coerce`` keyword must be explicitly used.
+
+  .. code-block:: python
+
+    In [2]: df = pd.DataFrame({'i': ['1','2'], 'f': ['apple', '4.2']})
```

Comment:
show the results of this code (no need to show the original); you may also way to show
``df.convert_objects(dates=False, numeric=True, coerce=True).dtypes``